CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## 1.12.1 - 2020-02-10
 ### Fixed
 - Fixed issue where client did not generate functions for deprecated API endpoints. (#353)
+
+- Fixed a bug related to duplicating parent job parameters when using `civis.parallel.infer_backend_factory`. (#363)
```

Comment:
Request: This should go in the "unreleased" section of the changelog. This section is for changes that went out with the most recent release.

Reply:
oops!